src/main/kotlin/com/team4099/robot2023/commands/manipulator/OuttakeConeCommand.kt

```diff
@@ -0,0 +1,29 @@
+package com.team4099.robot2022.commands.intake
+
+import com.team4099.robot2023.config.constants.ManipulatorConstants
+import com.team4099.robot2023.subsystems.manipulator.Manipulator
+import edu.wpi.first.wpilibj2.command.CommandBase
+import org.littletonrobotics.junction.Logger
+
+class OuttakeConeCommand(val manipulator: Manipulator) : CommandBase() {
+  init {
+    addRequirements(manipulator)
+  }
+
+  override fun initialize() {
+
+    manipulator.rollerState = ManipulatorConstants.RollerState.CONE_OUT
+  }
+
+  override fun execute() {
+    Logger.getInstance().recordOutput("ActiveCommands/ReverseConeCommand", true)
```

Comment:
Make sure the name of what you log matches the name of the command, otherwise it will make debugging unnecessarily difficult